test/command.mjs

```diff
@@ -16,7 +18,7 @@ async function wait() {
 
 describe('Command', () => {
   beforeEach(() => {
-    yargs.getInternalMethods().reset();
+    yargs().getInternalMethods().reset();
```

Comment:
Is this needed? It looks like it creates a yargs instance, calls a method on it, and abandons it to the garbage collection. So surplus to requirements now there is not a singleton in play.

Reply:
Found it was deleted in another test file, so probably stale.